conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -67,17 +67,15 @@ export UPLOAD_PACKAGES="${UPLOAD_PACKAGES:-True}"
            -v "${RECIPE_ROOT}":/home/conda/recipe_root:rw,z \
            -v "${FEEDSTOCK_ROOT}":/home/conda/feedstock_root:rw,z \
            -e CONFIG \
-           -e BINSTAR_TOKEN \
            -e HOST_USER_ID \
            -e UPLOAD_PACKAGES \
            -e GIT_BRANCH \
            -e UPLOAD_ON_BRANCH \
            -e CI \
-{%- if conda_forge_output_validation %}
            -e FEEDSTOCK_NAME \
```

Comment:
This variable is visible in non-docker builds. I don't see why we should not always set it.

Reply:
Yup seems ok.